### ISSUE TITLE
Drop hot key customization options

### ISSUE THREAD

**Issue by rugk** [keepassxreboot/keepassxc-browser#389]
You currently provide a custom way to adjust hot keys of your extension.

Starting with Firefox 66 [Firefox will provide a built-in way of doing so](https://bugzilla.mozilla.org/show_bug.cgi?id=1303384) ([German article](https://www.soeren-hentzschel.at/firefox/tastatur-kommandos-von-erweiterungen-koennen-ab-firefox-66-geaendert-werden/)) and thus you can IMHO drop your own customization options in `/options/shortcuts.html`.

AFAIK other browsers like Chrome/ium already support that customization as a built-in feature too, so there is no need to reimplement that in your extension anymore.

**Comment by varjolintu** [comment]
Yes, I'm aware of that. When doing to feature there was no public roadmap for the feature. It's good to know official support is soon coming.

**Comment by rugk** [comment]
Yes, of course you do not have to do it now. Or you maybe detect the FF version and show/hide the feature programmatically…

**Comment by varjolintu** [comment]
I'll make the fix before Firefox 66 is released :) The version comparison function already exists.

**Comment by varjolintu** [comment]
I'm still keeping the feature visible for Firefox 66 because for some reason Mozilla did not implement a URL that could go straight to the shortcut settings. Instead you must go to `about:addons`, then click the gear icon and open the page manually. Not so user friendly.

**Comment by rugk** [comment]
Okay, opened an issue for that: https://bugzilla.mozilla.org/show_bug.cgi?id=1538451

**Comment by rugk** [comment]
Now status in the issue:

> "design-decision-approved"

As far as I see, this looks good. It has prio 5, however, so they accept contributions on this issue.

**Comment by varjolintu** [comment]
Dropping by to comment that at least there's something happening in the ticket after a few years.

**Comment by varjolintu** [comment]
Finally!
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/commands/openShortcutSettings
